PseudoGantry/lib/PseudoGantry/PseudoEncoder.cpp

```diff
@@ -0,0 +1,9 @@
+#include "PseudoEncoder.h"
+
+void set_up_encoder(PseudoEncoder *encoder, void (*isr_encoder)(void)) 
+{
+    pinMode(encoder->motor_pulse_pin, INPUT_PULLUP);
+    pinMode(encoder->channel_a_out, OUTPUT);
+    digitalWrite(encoder->channel_a_out, LOW);
+    attachInterrupt(digitalPinToInterrupt(encoder->motor_pulse_pin), isr_encoder, CHANGE);
```

Comment:
`isr_encoder` is attached to the `motor_pulse_pin`
Might be better to name it `isr_motor_pulse`?